data/toasterson.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
The filename needs to match your username, including case

Reply:
Oh, fixed.